### PR TITLE
Adding PCI 3.2.1 and CIS 1.3.0 Frameworks for AWS

### PR DESCRIPTION
**Pull request by cbc506**

### CODE REVIEW

**Comment by yi2020** [comment]
@cbc506 you have a conflict, can you correct it?

**Review by Viperain — APPROVED** [review]

**Comment by cbc506** [comment]
> @cbc506 you have a conflict, can you correct it?

I just resolved it, please check.

**Comment by yi2020** [comment]
@Viperain tests are failing, can you assist?

**Comment by cbc506** [comment]
> @Viperain tests are failing, can you assist?

The error suggests that a rule is missing in the metadata file so I diff'd it and found the missing rule, will add changes and commit.

**Comment by cbc506** [comment]
> @Viperain tests are failing, can you assist?

Just added code for a missing rule metadata but it's still failing, please assist.

File: cloudrail/knowledge/rules/aws/aws_rules_metadata.yaml [hunk 1]
```diff
@@ -3701,9 +4302,34 @@ rules_metadata:
...
+  security_layer: logging
+  severity: major
+  compliance:
+    "PCI DSS":
+      "3.2.1": 
+        - "10.1"
+        - "10.2"
+=======
   
 - rule_id: ensure_cloudtrail_trail_exists
```
Comment (yi2020) [inline cloudrail/knowledge/rules/aws/aws_rules_metadata.yaml:4331]: @cbc506 you're missing compliance info for the rule you added

**Comment by yi2020** [comment]
@cbc506 the metadata file had lines from the merge conflict (see the commit I just made).
Also note my comment on compliance info missing from the rule you added

**Pull request merged**